src/client/voice/VoiceConnection.js

```diff
@@ -504,7 +508,7 @@ class VoiceConnection extends EventEmitter {
     }
   }
 
-  play() {} // eslint-disable-line no-empty-function
+  play() { } // eslint-disable-line no-empty-function
```

Comment:
```suggestion
  play() {} // eslint-disable-line no-empty-function
```
Nit: formatting should stay the same as it's like that throughout d.js

Reply:
Aggg, that's VSCode (I'm just using the default setup), fixed (x2)